NUITizenGallery/Examples/ButtonTest/ButtonTest8.cs

```diff
@@ -0,0 +1,539 @@
+﻿using System.Collections.Generic;
+using Tizen;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Components;
+
+namespace NUITizenGallery
+{
+    internal class CommonResource
+    {
+        public static string GetResourcePath()
+        {
+            return Tizen.Applications.Application.Current.DirectoryInfo.Resource + "images/"; // TODO...
+        }
+    }
+
+    internal class ButtonPage8 : ContentPage
+    {
+        private View root;
+        private View parent1;
+        private View parent2;
+        private View parent3;
+        Button textButton;
+        Button iconButton;
+
+        Button iconTextButton;
+
+        Button utilityBasicButton;
+        Button utilityServiceButton;
+        Button utilityToggleButton;
+        Button utilityOvalButton;
+
+        Button familyBasicButton;
+        Button familyServiceButton;
+        Button familyToggleButton;
+        Button familyOvalButton;
+
+        internal ButtonPage8(Window window)
+        {
+            window.KeyEvent += Window_KeyEvent;
+					
+            WidthSpecification = LayoutParamPolicies.MatchParent;
+            HeightSpecification = LayoutParamPolicies.MatchParent;
+
+            AppBar = new AppBar()
+            {
+                Title = "Photo Slide Sample",
+            };
+						
+            root = new View()
+            {
+                Size = new Size(1920, 1080),
```

Comment:
could you change this as using Layout ?
fixed size can not be tested in various targets.